device-types/QNAP/TS-832PXU-RP.yaml

```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: QNAP
+model: TS-832PXU-RP
+slug: qnap-ts-832pxu-rp
+part_number: TS-832PXU-RP
+u_height: 2
+airflow: front-to-rear
+front_image: true
+rear_image: true
+is_full_depth: true
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
+  - name: Disk 5
+  - name: Disk 6
+  - name: Disk 7
+  - name: Disk 8
+module-bays:
+  - name: PCIe1
+    position: PCIe1
+comments: '[TS-832PXU-RP | Hardware Specs | QNAP](https://www.qnap.com/en/product/ts-832pxu-rp/specs/hardware)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 70
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 70
```

Comment:
As this has removable PSUs they should be modelled under `module-bays` and then added the required PSU modules if they don't already exists in the `module-bays/QNAP/`

Reply:
Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be